### PR TITLE
Missing check-in docs

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Brings missing content from wiki about Check-in. Also moves the VOMS into VOs

---

### CODE REVIEW

**Review by andrea-manzi — APPROVED** [review]
👍 👍

File: content/en/users/check-in/faq/_index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+CheckIn service nor for the Identity provider. The connection needs to
+be implemented in the hub and spoke IdP Proxy.
+
+One example of such federation is SURFconext, the national IdP
+federation for research and education in the Netherlands operated by
+SURFnet. If you are using credentials from a Dutch IdP in eduGAIN, you
+or your IdP administrators need to request the connection. The following
+steps will lead you to perform the connection:
+
+-   Connect to [SURFconext
```
Comment (gwarf) [inline content/en/users/check-in/faq/_index.md:27]: Please fix the style of items list (only one space after the dash)

File: content/en/users/check-in/faq/_index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    should be a \"Connect\" button, this will create a service ticket
+    and the SURFconext team will make the connection active.
+-   After you received confirmation that the \"EGI AAI Service provider
+    proxy\" is accessible, you will be able to login in CheckIn
+
+## Authentication error with ADFS-based Identity Providers
+
+### Why do I get the error below after successfully authenticating at my Home IdP?
+
+```
```
Comment (gwarf) [inline content/en/users/check-in/faq/_index.md:43]: Would be good to have a file type for the excerpt, super-linter would complain.
But not sure anything is relevant there/if there is some kind of raw/text type.

File: content/en/users/check-in/faq/_index.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+opensaml::FatalProfileException at (https://aai.egi.eu/registry.sso/SAML2/POST)
+SAML response reported an IdP error.
+Error from identity provider:
+Status: urn:oasis:names:tc:SAML:2.0:status:Responder
+```
+
+The Responder error status is typically returned from ADFS-based IdP
+implementations (notably Microsoft ADFS 2.0 and ADFS 3.0) that cannot
+properly handle Scoping elements (see
+<https://docs.microsoft.com/en-za/azure/active-directory/develop/active-directory-single-sign-on-protocol-reference#scoping>).
```
Comment (gwarf) [inline content/en/users/check-in/faq/_index.md:53]: This is not a proper link.

File: content/en/users/check-in/vos/expressing-vo-information/_index.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+The general form of the eduPersonEntitlement string is:
+
+`<NAMESPACE>:group:<VO>[:<GROUP>*][:role=<ROLE>]#<GROUP-AUTHORITY>` 
+
+As a result, an eduPersonEntitlement string informing the Service
+Provider that the user has the role Associate in the vo.example.eu 
+VO (modelled as a COU) is:
+
+`urn:mace:egi.eu:group:vo.example.eu:role=associate#aai.egi.eu`
+
```
Comment (gwarf) [inline content/en/users/check-in/vos/expressing-vo-information/_index.md:27]:
```suggestion
```

File: content/en/users/check-in/vos/expressing-vo-information/_index.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+for each different Role and for the Affiliation. For example, the CO
+Person from above is affiliated as Member to the VO vo.example.eu
+and has been assigned the role of an Associate. This will generate
+two entitlements as:
+
+`urn:mace:egi.eu:group:vo.example.eu:role=associate#aai.egi.eu`
+
+`urn:mace:egi.eu:group:vo.example.eu:role=member#aai.egi.eu`
+
+
```
Comment (gwarf) [inline content/en/users/check-in/vos/expressing-vo-information/_index.md:51]:
```suggestion
```

File: content/en/users/workload-manager/_index.md [hunk 6]
```diff
@@ -135,27 +135,23 @@ Team on shift.
...
-[EUGridPMA-certified](http://www.eugridpma.org/)
-authority you can use it for this tutorial. Otherwise refer to the information
-available in this
-[section](../check-in/voms/#get-a-certificate),
-to obtain a certificate.
-Your certificate may take a few days to be delivered, so please ask for your
-certificate well in advance and in any case, before the tutorial starts.
+DIRAC uses [X.509 certificates](../check-in/vos/voms/) to identify and authenticate
+users. These certificates are delivered to each individual by trusted
+certification authorities.\
```
Comment (gwarf) [inline content/en/users/workload-manager/_index.md:140]: Why this last `\` ?

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks, a few comments, please at least fix the list items spacing.

File: content/en/users/workload-manager/_index.md (hunk 6 above)
Comment (enolfc) [reply content/en/users/workload-manager/_index.md:140]: 🤷 , removing it

File: content/en/users/check-in/faq/_index.md (hunk 3 above)
Comment (enolfc) [reply content/en/users/check-in/faq/_index.md:53]: you mean the markdown syntax is not the one we should use or that the link is not the one that should be used?

File: content/en/users/check-in/faq/_index.md (hunk 2 above)
Comment (enolfc) [reply content/en/users/check-in/faq/_index.md]: using plaintext

File: content/en/users/check-in/faq/_index.md (hunk 3 above)
Comment (gwarf) [reply content/en/users/check-in/faq/_index.md:53]: The syntax no?
Linters seems to prefer to have `[]()`

Comment (enolfc) [reply content/en/users/check-in/faq/_index.md]: fixed

**Review by gwarf — APPROVED** [review]

**Comment by gwarf** [comment]
@CatalinCondurache , @vardizzo-lab it's ready for you!

**Comment by gwarf** [comment]
@enolfc in fact it seems `markdown-link-check` found an issue: https://github.com/EGI-Foundation/documentation/pull/121/checks?check_run_id=1358152008#step:4:55

**Comment by enolfc** [comment]
> @enolfc in fact it seems `markdown-link-check` found an issue: https://github.com/EGI-Foundation/documentation/pull/121/checks?check_run_id=1358152008#step:4:55
> 
> I'm also expecting the linter to complain for some stuff (that I fixed in the other PR I made but that I closed...)

I think it's better to fix that into a separate PR, merge and then rebase this one... on my way

**Comment by enolfc** [comment]
> > @enolfc in fact it seems `markdown-link-check` found an issue: https://github.com/EGI-Foundation/documentation/pull/121/checks?check_run_id=1358152008#step:4:55
> > I'm also expecting the linter to complain for some stuff (that I fixed in the other PR I made but that I closed...)
> 
> I think it's better to fix that into a separate PR, merge and then rebase this one... on my way

Nevermind... it's this very same PR

**Comment by enolfc** [comment]
@vardizzo-lab , @CatalinCondurache could you take a look? This is now passing all tests

File: content/en/users/check-in/vos/voms/_index.md [hunk 7]
```diff
@@ -285,6 +285,7 @@ voms-proxy-init --voms <name of the vo> --rfc
 
 See for example, using `fedcloud.egi.eu` VO:
 
+<!-- markdownlint-disable line-length -->
 ```{.console}
```
Comment (gwarf) [inline content/en/users/check-in/vos/voms/_index.md:289]: @enolfc you didn't touch it right now, but the code fence should more be something like;
```markdown
```sh
echo $PLOP
```

Comment (gwarf) [reply content/en/users/check-in/vos/voms/_index.md:289]: I think `{.console}` is a legacy from the restructured doc

Comment (enolfc) [reply content/en/users/check-in/vos/voms/_index.md:289]: there is a lot of that:
```
$ git grep -l console content/
content/en/providers/cloud-compute/configuration/_index.md
content/en/providers/cloud-compute/opennebula/_index.md
content/en/providers/cloud-compute/openstack/_index.md
content/en/providers/cloud-compute/validation/_index.md
content/en/users/check-in/vos/voms/_index.md
content/en/users/cloud-compute/auth/_index.md
content/en/users/cloud-compute/ec3/ecas.md
content/en/users/cloud-compute/ec3/htc.md
content/en/users/cloud-compute/faq/_index.md
content/en/users/cloud-compute/openstack/_index.md
content/en/users/cloud-compute/storage/_index.md
content/en/users/cloud-compute/vmi/_index.md
content/en/users/cloud-container-compute/docker.md
content/en/users/data-transfer/WebFTS/_index.md
content/en/users/datahub/_index.md
```

I will fix the check-in ones here

Comment (gwarf) [reply content/en/users/check-in/vos/voms/_index.md]: Yes basically everything that was imported from restructuredtext have some dirty leftovers that went though the review as mdl wasn't strict enough (and it was lots of content).

**Pull request merged**